src/OrchardCore.Modules/OrchardCore.Media/OrchardHelperExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace OrchardCore.Media
+{
+    /// <summary>
+    /// Provides an extension methods for <see cref="IOrchardHelper"/>.
+    /// </summary>
+    public static class OrchardHelperExtensions
+    {
+        private static readonly HashSet<string> _imageExtensions = new HashSet<string>() { "bmp", "gif", "jpeg", "jpg", "png", "tiff", "webp" };
```

Comment:
Why we don't have `.`, so we can check if the list contains the extension, instead of stripping the `.` whenever we call this function